src/main/java/nextstep/controller/LadderGameController.java

```diff
@@ -1,19 +1,32 @@
 package nextstep.controller;
 
-import nextstep.domain.Lines;
+import nextstep.domain.*;
 import nextstep.view.InputView;
 import nextstep.view.OutputView;
 
-import java.util.List;
-
 public class LadderGameController {
 
     public static void main(String[] args) {
-        List<String> userNames = InputView.inputUserNames();
+        Usernames usernames = new Usernames(InputView.inputUserNames());
+        InputResults inputResults
+                = new InputResults(InputView.inputLadderResult(), usernames.size());
         int ladderHeight = InputView.inputLadderHeightMax();
 
-        Lines lines = new Lines(userNames.size(), ladderHeight);
-        OutputView.outputUserNames(userNames);
+        Lines lines = new Lines(usernames.size(), ladderHeight);
+        OutputView.outputUserNames(usernames.getUsernames());
         OutputView.outputLadder(lines);
+        OutputView.outputLadderResult(inputResults.getResults());
+
+        Ladder ladder = new Ladder(lines, usernames.size());
+        LadderResult ladderResult = new LadderResult(ladder, usernames, inputResults);
+
+        while (true) {
+            Username name = new Username(InputView.inputUsernameForResult());
+            OutputView.outputExecutionResult(ladderResult.getLadderResult(name));
+
+            if (name.equals(new Username("all"))) {
```

Comment:
매번 같은 유저를 생성하는 것 보다 
```suggestion
            if (name.equals(Username.ALL)) {
```
상수로 사용해봐도 괜찮을 것 같아요!